unicodecsv/test.py

```diff
@@ -745,6 +745,20 @@ def test_read_semi_sep(self):
         self.assertEqual(reader.next(), {"1": '1', "2": '2', "3": 'abc',
                                          "4": '4', "5": '5', "6": '6'})
 
+    def test_empty_file(self):
+        fd, name = tempfile.mkstemp()
```

Comment:
This can be reproduced with StringIO:

```
unicodecsv.DictReader(StringIO.StringIO())
```

That avoids the need to create the tempfile on disk.